zulipterminal/ui_tools/views.py

```diff
@@ -967,27 +968,45 @@ def __init__(self, controller: Any, msg: Any) -> None:
         super().__init__(controller, self.widgets, 'MSG_INFO')
 
     def build_pile(self, snapshot: Dict[str, Any], tag: str='',
-                   ) -> Any:
+                   author_prefix: str='Edited') -> Any:
         timestamp = time.ctime(snapshot['timestamp'])[:-5]
+        author = self.get_author(snapshot['user_id'])
         topic = snapshot['topic']
 
         if 'prev_topic' in snapshot and topic != snapshot['prev_topic']:
             topic += ' (updated from {})'.format(snapshot['prev_topic'])
+            if 'prev_content' in snapshot:
```

Comment:
We seem to show the fact that the topic has been updated, in both the title and the body here. We show it in `updated from {}` (Title) and show who it was updated by as well. Do you think sticking with `Edited by X` might be simpler here, even for a topic update? Might simplify things a bit. Might be a discussion to have on the stream.